changed camelCase params to lower case

1099511627776 opened the pull request:
changed camelCase param in settings to lower case according to comment in https://github.com/tinymce/tinymce/pull/2581 this pull request. 
check trough travis-ci https://travis-ci.org/1099511627776/tinymce

fyrkant commented:
I am closing this pull request due to old age and merge conflicts. If you still think this is relevant and something that should be merged, fix the merge conflicts and leave a comment and I can open it up again. Thank you for the contribution!

1099511627776 commented:
Well I see no need to open this pull request again bacause I don't see your interest in this pull request.

As you can see I've fixed your issues metioned in https://github.com/tinymce/tinymce/pull/2581 and also added link to travis-ci tests https://travis-ci.org/1099511627776/tinymce for more than a year but you did not even look at them. Of course today this issue has many merge conflicts but I don't think It's my fault because when this pull request was opened (on 2 Apr 2016) there were no conflicts.

So if you are interested in implementing this functionality than I suggest you to reopen this pull request and have a further dialog more frequently that once per YEAR
